azurerm/resource_arm_user_assigned_identity.go

```diff
@@ -107,6 +112,12 @@ func resourceArmUserAssignedIdentityRead(d *schema.ResourceData, meta interface{
 		}
 	}
 
+	if props := resp.IdentityProperties; props != nil {
+		if clientId := props.ClientID; clientId != nil {
```

Comment:
This could be move up into the above if statement:

```go
   if props := resp.IdentityProperties; props != nil {
 		if principalId := props.PrincipalID; principalId != nil {
 			d.Set("principal_id", principalId.String())
		}

        if clientId := props.ClientID; clientId != nil {
			d.Set("client_id", clientId.String())
		}
	}

```